### PR TITLE
Bump csi driver version

### PR DESCRIPTION
**Pull request by ritazh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -136,12 +140,6 @@ To verify that Secrets Store CSI Driver has started, run:
...
 
-$ kubectl --namespace=dev get pods -l "app=secrets-store-csi-driver"
-NAME                                     READY     STATUS    RESTARTS   AGE
-csi-secrets-store-attacher-0                  1/1       Running   0          43s
-csi-secrets-store-secrets-store-csi-driver-f8lw6   2/2       Running   0          43s
-csi-secrets-store-secrets-store-csi-driver-hj445   2/2       Running   0          43s
-
 ```
 
 <details>
```
Comment (aramase) [inline README.md]: I think we need to add steps here for installing provider using deployment yamls?

**Comment by ritazh** [comment]
@aramase I've disabled both providers in values.yaml as I think this should be explicitly defined by the operator as oppose to silently defaults to something. WDYT?

**Comment by aramase** [comment]
@ritazh agreed!

File: README.md [hunk 2]
```diff
@@ -271,6 +291,29 @@ End-to-end tests automatically runs on Travis CI when a PR is submitted. If you
...
+  ```bash
+  kubectl get pod -o wide
+  # find the secrets store csi driver pod running on the same node as your application pod
+
+  kubectl logs csi-secrets-store-secrets-store-csi-driver-7x44t secrets-store
+  ```
+- To troubleshoot issues with the provider component, you can look at logs from the `provider-log` container of the provider pod running on the same node as your application pod:
+  ```bash
+  kubectl get pod -o wide
+  # find the secrets store csi driver pod running on the same node as your application pod
```
Comment (aramase) [inline README.md]: nit: find the secrets store csi driver provider pod running on the same node as your application pod

**Pull request merged**